composer/lib/dependabot/composer/file_fetcher.rb

```diff
@@ -90,17 +126,24 @@ def path_dependencies
           end
       end
 
+      def artifact_sources
```

Comment:
seems only the 'type' param differs between both methods? Deferring to your decision though

Reply:
This seems more like a Ruby pattern of having `artifact_sources` and `path_sources` rather than `sources(type: "artifact")` which is why I went with this.

Reply:
Thanks; TIL; I didn't know that!